code/game/objects/items/weapons/mine.dm

```diff
@@ -30,12 +30,17 @@
 /obj/item/weapon/mine/ignite_act()
 	explode()
 
+/obj/item/weapon/mine/ex_act()
+	explode()
+
+/obj/item/weapon/mine/bullet_act()
+	explode()
+
 /obj/item/weapon/mine/proc/explode()
 	var/turf/T = get_turf(src)
+	qdel(src)
 	explosion(T,explosion_d_size,explosion_h_size,explosion_l_size,explosion_f_size)
 	fragment_explosion(T, spread_radius, fragment_type, num_fragments, null, damage_step)
```

Comment:
```suggestion
	explosion(T,explosion_d_size,explosion_h_size,explosion_l_size,explosion_f_size)
	fragment_explosion(T, spread_radius, fragment_type, num_fragments, null, damage_step)
	qdel(src)
```

Reply:
No, bad. Tried it. Caused infinite explosions as the explosion would call ex_act on the mine.